tests/env/start-swift-ceph.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
is this file a copy of https://github.com/owncloud/objectstore/blob/master/tests/env/start-s3-ceph.sh ?